### ISSUE TITLE
MOL-Prometheus IIIM difficulty Reaching Polar Orbit in JNSQ

### ISSUE THREAD

**Issue by CXG-2827** [CobaltWolf/Bluedog-Design-Bureau#873]
Built the craft per:
https://github.com/friznit/Unofficial-BDB-Wiki/wiki/MOL

It can get to orbit at zero inclination, but polar it falls short. If you play with life support/kerbalism the added mass of food/water/O2/etc. to sustain 40 days shaves another ~200 Delta-V.

The MOS-NL10 "Dorian" Recon system shows a mass of 4.5, which is about the same as 3 of the lab segments (same length roughly too). Though the recon section is basically a dobsonian-style telescope, so the mass is mostly the shell, plus the glass of the primary/secondary mirrors. Might be ok to take that weight down to ~1.75-2T from the current 4.5T.

**Comment by jsolson** [comment]
No Kerbalism here, but I just put it into a 153x57 at 85 deg orbit with 531 m/s left in the Titan second stage, and circularized with the MOL thrusters with 300+ m/s rcs dv remaining. It's got literally a ton of excess monopropellent capacity. How much weight is Kerbalism adding?

![Untitled](https://user-images.githubusercontent.com/16825376/79036470-87178d80-7b96-11ea-895e-2f2160fc3ccf.png)

**Comment by CXG-2827** [comment]
Any chance of getting a craft file so I can check for discrepancies on my end? I couldnt find one for the Titan IIIM, so there could be that I goofed it up.

The RCS MOL thrusters barely changes the velocity when engaged. I disabled the "Meduci" RCS thrusters since the thrust was clipping through the lab module, and I assumed it was not intended to be used.

Kerbalism adds the option to include radiation shielding to crewable parts, so in the instance of the MOL lab module, toggling max shielding adds 1 ton. My mass is 174.747T, but I did tweak the Recon module CFG from 4.5T -> 2T
![Capture](https://user-images.githubusercontent.com/5134828/79036801-479a7280-7b91-11ea-8212-dc254ceee00b.PNG)

**Comment by jsolson** [comment]
Remove the .txt extention.

[Titan IIIM.craft.txt](https://github.com/CobaltWolf/Bluedog-Design-Bureau/files/4464699/Titan.IIIM.craft.txt)

**Comment by CXG-2827** [comment]
Thanks. I gave your craft a try and I was able to get a 122x67 87 deg inclination with 435 Delta-V remaining in the 2nd stage. Flown manually, no mechjeb.

Flew with the Kerbalism consumables and radiation shielding, added ~2.4T. I was able to get 117x55 with a depleted 2nd Stage, but the RSC was able to circularize easily.

I think one of the primary contributing issues was that I did not have auto-strut enabled, so the wobbliness of the rocket lost a fair bit of Delta-V. I forgot auto-strut existed.

So I figure it really isnt much of a concern for Kerbalism users if they remember auto-strut. Though might still be worth considering the recon system mass reduction by a ton. I'm fine with editing my own CFGs, so I have no problem if you guys think 4.5 is perfectly fine.

Thanks again for the help.
